### PR TITLE
feat: add diamond blastx tool

### PR DESCRIPTION
**Pull request by jherr-dev**
Adds the blastx mode for diamond. The test will be updated after the backend is merged.

### CODE REVIEW

**Review by bcai2 — APPROVED** [review]
LGTM if works

File: tests/test_diamond.py [hunk 1]
```diff
@@ -27,3 +27,22 @@ def test_diamond_blastp_standard():
     )
 
     assert hash.unordered(output_file_path) == DEFAULT_OUTPUT_HASH
+
+
+@pytest.mark.skip(reason="need to find faster test file then it's hash")
```
Comment (lebovic) [inline tests/test_diamond.py:32]: 20min is okay to include in the meantime ¯\_(ツ)_/¯

File: toolchest_client/tools/diamond.py [hunk 2]
```diff
@@ -32,3 +32,27 @@ def __init__(self, inputs, output_name, output_path, tool_args, **kwargs):
...
+            inputs=inputs,
+            min_inputs=1,
+            max_inputs=1,
+            database_name="diamond_blastx_standard",
+            database_version="1",
+            parallel_enabled=False,
+            output_type=OutputType.GZ_TAR,
+            output_path=output_path,
+            output_is_directory=False,
+            **kwargs,
```
Comment (lebovic) [inline toolchest_client/tools/diamond.py:57]: Should include the expected output file names

**Review by lebovic — APPROVED** [review]
One flag, otherwise looks good!

**Pull request merged**